.clang-format

```diff
@@ -7,6 +7,7 @@ BreakBeforeBraces: Attach
 SpaceAfterCStyleCast: false
 Cpp11BracedListStyle: false
 IndentCaseLabels: true
+IndentPPDirectives: AfterHash
```

Comment:
can also be BeforeHash

Reply:
I like it :) . Though prefer `BeforeHash` myself.

Reply:
if we do this I would want before hash

Reply:
I'm an AfterHash enjoyer 😨

I'm not really sure if this is worthwhile, at least currently, considering most nested preprocessor directives are in headers and we don't format headers (yet?)

Reply:
this came up because of a case in another pr: https://github.com/zeldaret/oot/pull/1670#discussion_r1472156367
This will only become more common as we are working on retail

Reply:
we have a bunch of nested directives coming in along the lines of `#if debug #define STMT mydbgstmt #else #define STMT false #endif`

Reply:
I guess this is going to be all preference at the end of the day, but for me I am always used to seeing, for example, `#define` grouped together, seeing the word "define" far away from the hash makes it read not as a directive for me anymore